update.sh

```diff
@@ -30,7 +30,7 @@ update_gitignore () {
 
 # Check whether it is safe for the script to run
 check () {
-	if ! grep -q '"govuk-prototype-kit"' package.json 2> /dev/null; then
+	if ! grep -q '"govuk-prototype-kit"\|"GOVUK prototyping app in Express"' package.json 2> /dev/null; then
```

Comment:
I'm guessing this is needed because [the name was changed](https://github.com/alphagov/govuk-prototype-kit/commit/7cff511b3726cc6d3c34b206c3e4320832cc6e62) in v9.1.0?

If so, `govuk-prototype-kit` is the current _name_ in current package.json, whereas "GOVUK prototyping app in Express" is the old _description_.

It might make sense to use the old name instead to be consistent?

```suggestion
	if ! grep -q '"govuk-prototype-kit"\|"express-prototype"' package.json 2> /dev/null; then
```

It may not be worth fixing now, but it would have been good if this change had been made in a separate commit (or possibly a separate PR, given it's unrelated to #1286), and with a description that explained the problem and how it solved it.